### PR TITLE
docs for service provider metadata file

### PR DESCRIPTION
**Pull request by evanluc**
Documentation for getting the metadata file for a given service provider.

### CODE REVIEW

**Review by lamminade — APPROVED** [review]
lgtm

File: source/includes/administration/_service_providers.md [hunk 1]
```diff
@@ -304,3 +304,57 @@ curl "https://cloudmc_endpoint/rest/service_providers/f9dea588-d7ab-4f42-b6e6-4b
 ```
 
 Delete a specific service provider. 
+
+<!-------------------- GET SERVICE PROVIDER METADATA -------------------->
+
+
+#### Get service provider metadata file
+
+`GET /service_providers/:id`
```
Comment (GuillaumeVialle) [inline source/includes/administration/_service_providers.md]: isn't it `GET /service_providers/:id/metadata` ?

Comment (evanluc) [reply source/includes/administration/_service_providers.md]: :0 good eye
sure is

Comment (evanluc) [inline source/includes/administration/_service_providers.md]:
```suggestion
`GET /service_providers/:id/metadata`
```

Comment (evanluc) [reply source/includes/administration/_service_providers.md]: https://github.com/cloudops/cloudmc-api-docs/pull/184/commits/60494063e44a79425684c7894d75e589e93a338f

**Review by GuillaumeVialle — APPROVED** [review]

**Pull request merged**